### ISSUE TITLE
server-side messages repeatedly shown by command line tools

### ISSUE THREAD

**Issue by pdawyndt** [unipept/unipept#335]
`@tvwillem` @bmesuere I keep getting the warning

```
Unipept 0.4.0 is released!
```

although I have seen this warning before and I'm already working with version 0.4.1. Can you make sure that:
- messages are shown only once (possibly they are postponed until running an interactive session, but as a user I don't want to see the same message over and over again)
- messages are shown only if I run a specific version of the command line tools, e.g. if I run version 0.4.1, I don't want to see messages that are specific for older versions

Can we show an archive of Unipept API-related messages on the Unipept website, for users that may have missed some of the messages?

_[Original issue](https://github.ugent.be/unipept/unipept/issues/335) by @pdawyndt on Mon May 19 2014 at 09:22._
_Closed by @bmesuere on Wed Apr 22 2015 at 14:24._

**Comment by bmesuere** [comment]
This message was just an example of the functionality, I have now adapted it to not display if you're using the latest version.


_[Original comment](https://github.ugent.be/unipept/unipept/issues/335#issuecomment-9859) by ghost on Mon May 19 2014 at 22:31._

**Comment by bmesuere** [comment]
Still an issue. Latest version is now a config value: https://github.ugent.be/bmesuere/unipept/blob/develop/config/application.rb#L58


_[Original comment](https://github.ugent.be/unipept/unipept/issues/335#issuecomment-10513) by @bmesuere on Thu Aug 14 2014 at 10:56._

**Comment by bmesuere** [comment]
This was fixed a while ago


_[Original comment](https://github.ugent.be/unipept/unipept/issues/335#issuecomment-16454) by @bmesuere on Wed Apr 22 2015 at 14:24._
